### PR TITLE
Pugbomb fix

### PR DESCRIPTION
**Pull request by PhlipPhlops**
Edited error message when non-number is called. Issue #51

### CODE REVIEW

File: scripts/pugbomb.py [hunk 1]
```diff
@@ -5,9 +5,15 @@
 
 def pugbomb(response, args):
     #gets the number
-    num = int(args[0])
+    try:
+        num = round(int(args[0]))
+    except:
+        return "`Please enter a number next time`"
```
Comment (mhijazi1) [inline scripts/pugbomb.py:11]: I might be missing something, but is there a reason you nead `"` and `'` in the same string?

Comment (PhlipPhlops) [reply scripts/pugbomb.py:11]: Those are the backwards tick marks that change the format on slack

Comment (mhijazi1) [reply scripts/pugbomb.py:11]: 👍

**Review by mhijazi1 — APPROVED** [review]

File: scripts/pugbomb.py (hunk 1 above)
Comment (ghost) [reply scripts/pugbomb.py:11]: Keep in mind ' and ` are different

Comment (mhijazi1) [reply scripts/pugbomb.py:11]: Yeah, it looked like `'` to me, didn't notice it was ``` till after

**Pull request merged**